lib/beaker/host/unix/pkg.rb

```diff
@@ -120,6 +120,34 @@ def install_package(name, cmdline_args = '', version = nil, opts = {})
     end
   end
 
+  # Extract RPM command's proxy options from URL
+  #
+  # @param [Url] url A URL of form http://host:port
+  #
+  def extract_rpm_proxy_options(url)
```

Comment:
Interestingly, it looks like it's not really possible to use rspec to test private methods https://www.ruby-forum.com/topic/197346

I am a bit surprised; I think you should be able to test private methods.

Anyhow the workaround I have chosen is to mark the method as private in the documentation only, although you may have a better suggestion.